tests/bindings-test/ApiDefinition.cs

```diff
@@ -376,14 +376,21 @@ interface ObjCBlockTester
 		void CallOptionalStaticCallback ();
 
 		[Static]
-		[Export ("callAssertMainThreadBlockRelease:")]
```

Comment:
You're removing an existing method: that's a breaking change, so we can't do that.